SportShop/MainWindow.xaml

```diff
@@ -0,0 +1,76 @@
+﻿<Window x:Class="SportShop.MainWindow"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        xmlns:local="clr-namespace:SportShop"
+        xmlns:local1="clr-namespace:SportShop.View"
+        xmlns:model ="clr-namespace:SportShop.Model"
+        xmlns:viewmodel ="clr-namespace:SportShop.ViewModel"
+        mc:Ignorable="d"
+        Title="Sport Shop" Height="600" Width="1000">
+
+    <Window.DataContext>
+        <viewmodel:ProductViewModel/>
+    </Window.DataContext>
+    <Window.Resources>
+        <DataTemplate x:Key="SportTemplate" DataType="{x:Type model:Product}">
+            <StackPanel Orientation="Horizontal">
+                <Image Source="{Binding Path=Image}" Width="100" Height="100"></Image>
+                <Label Content="{Binding Path=ProductName}"></Label>
+                <Label Content="{Binding Path=ProductModel}"></Label>
+                <Label Content="{Binding Path=Price}"></Label>
+            </StackPanel>
+        </DataTemplate>
+    </Window.Resources>
+
+    <Grid>
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="*"></ColumnDefinition>
+            <ColumnDefinition Width="*"></ColumnDefinition>
+        </Grid.ColumnDefinitions>
+
+        <Grid.RowDefinitions>
+            <RowDefinition Height="8*"></RowDefinition>
+            <RowDefinition Height="*"></RowDefinition>
+        </Grid.RowDefinitions>
+
+        <!--List of Sport Product-->
+        <StackPanel Grid.RowSpan="2" Grid.Column="0"
+                     Margin="20,30"
+                    Orientation="Vertical">
+            <Label Padding="5"
+                   FontSize="30">Sport Shop Products:</Label>
+            <ListBox x:Name="listOfProducts"
+                 Padding="5"
+                 BorderThickness="2"
+                 Height="Auto"
+                 
+                 ItemsSource="{Binding Products}"
+                 ItemTemplate="{StaticResource SportTemplate}"
+                 SelectedItem="{Binding SelectedProduct}" 
+                 ></ListBox>
```

Comment:
Please put closed tag on the same line as the latest property.

Reply:
Also it is better to use single tags where possible:
<Label Padding="5"
                   FontSize="30">Sport Shop Products: />
instead of
<Label Padding="5"
                   FontSize="30">Sport Shop Products:</Label>

Reply:
-1 to source code quality.